Week-5/ex 5.4/index.test.js

```diff
@@ -0,0 +1,16 @@
+const mathOperations = require('./index.js');
+
```

Comment:
You could have added negative number test case as well.

Reply:
added